project/Pack.scala

```diff
@@ -116,11 +118,21 @@ object Pack extends sbt.Build {
           "lib/storm" -> new ProjectsToPack(storm.id).exclude(streaming.id)
         ),
         packExclude := Seq(thisProjectRef.value.project),
+        packExcludeJars := Seq(
+          "akka-kryo-.*\\.jar",
+          "gs-collections-.*\\.jar",
+          "guava-.*\\.jar",
```

Comment:
There should be a guava artifact under `lib/yarn`, now after you exclude this when packing, does it still exist?